packages/odata-v2/src/request-builder/response-transformers.ts

```diff
@@ -4,13 +4,25 @@ import { DeSerializers } from '../de-serializers/de-serializers';
 import { Entity } from '../entity';
 import { getSingleResult, getCollectionResult } from './response-data-accessor';
 
+/**
+ * Convert the payload of the OData response to undefined.
+ * @param data - The OData payload.
+ * @param builderFn - The deserialization function for converting the payload.
+ * @returns The converted value.
+ */
 export function transformReturnValueForUndefined<ReturnT>(
   data: any,
   builderFn: (data: any) => ReturnT
 ): ReturnT {
   return builderFn(data);
 }
 
+/**
+ * Convert the payload of the OData response to an Entity by using given (De-)serializers.
+ * @param data - The OData payload.
+ * @param entityApi - Entity API that holds the (De-)serializers.
```

Comment:
[nth] Casing, but actually I think in other file you will find a different description for entityApi, maybe we should be consistent.

```suggestion
 * @param entityApi - Entity API that holds the (de-)serializers.
```